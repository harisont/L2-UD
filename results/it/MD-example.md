## Sentence 17-04_en-2:
L1 sentence | L2 sentence
--- | ---
« Non metta il **suo** **naso** in gli affari d' altri . » l' uomo ribatte . | « Non mette **suo** **naso** in gli affari d' altre . » l' uomo ribatte .


## Sentence 3-11_fr-3:
L1 sentence | L2 sentence
--- | ---
La donna ringraziava il **suo** **salvatore** con un abbraccio e chiudeva gli occhi . | La dona ringraziava **suo** **salvatore** con un braccio e chiusa le occhi .


