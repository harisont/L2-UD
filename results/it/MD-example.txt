17-04_en-2
3-11_fr-3
