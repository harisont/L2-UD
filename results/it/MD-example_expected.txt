17-04_en-2
3-11_fr-3
31-07_de-3
25-07_fr-1
21-08_en-2